scripts/windows/setup/Install-SecurityDaemon.ps1

```diff
@@ -42,13 +42,13 @@ function Install-SecurityDaemon {
     }
 
     if ((Test-EdgeAlreadyInstalled)) {
-        Write-Host ("`nIoT Edge appears to be already installed, exiting.") `
+        Write-Host ("`nIoT Edge is already installed, exiting.") `
```

Comment:
Can we add a message to run Uninstall-SecurityDaemon first, if users are looking to reinstall?